Lib/test/test_tarfile.py

```diff
@@ -3818,7 +3845,15 @@ def test_modes(self):
             pass
         new_mode = (os.stat(tmp_filename).st_mode
                     | stat.S_ISVTX | stat.S_ISGID | stat.S_ISUID)
-        os.chmod(tmp_filename, new_mode)
+        try:
+            os.chmod(tmp_filename, new_mode)
+        except OSError as err:
+            # gh-108948: FreeBSD fails with EFTYPE if sticky bit cannot be set,
+            # instead of ignoring it.
```

Comment:
"instead of ignoring it" is not clear. Maybe just remove it? Or explain that other platforms behave differently? You may mention that this is the behavior for non-root users.